src/odemis/util/test/transform_test.py

```diff
@@ -51,65 +51,65 @@
 
 RIGID_KNOWN_VALUES = [
     # (rotation, scale, shear, translation, [(x0, y0), (x1, y1), ...])
-    (0., 1., 0., T0, [(0., 0.), (1., 0.), (0., 1.)]),
-    (0., 1., 0., TX, [(1., 0.), (2., 0.), (1., 1.)]),
-    (0., 1., 0., TY, [(0., 1.), (1., 1.), (0., 2.)]),
-    (0., 1., 0., -TX, [(-1., 0.), (0., 0.), (-1., 1.)]),
-    (0., 1., 0., -TY, [(0., -1.), (1., -1.), (0., 0.)]),
-    (0., 1., 0., TXY, [(2., 3.), (3., 3.), (2., 4.)]),
-    (-ROT90, 1., 0., T0, [(0., 0.), (0., -1.), (1., 0.)]),
-    (-ROT90, 1., 0., TX, [(1., 0.), (1., -1.), (2., 0.)]),
-    (-ROT90, 1., 0., TY, [(0., 1.), (0., 0.), (1., 1.)]),
-    (-ROT90, 1., 0., -TX, [(-1., 0.), (-1., -1.), (0., 0.)]),
-    (-ROT90, 1., 0., -TY, [(0., -1.), (0., -2.), (1., -1.)]),
-    (-ROT90, 1., 0., TXY, [(2., 3.), (2., 2.), (3., 3.)]),
-    (ROT90, 1., 0., T0, [(0., 0.), (0., 1.), (-1., 0.)]),
-    (ROT90, 1., 0., TX, [(1., 0.), (1., 1.), (0., 0.)]),
-    (ROT90, 1., 0., TY, [(0., 1.), (0., 2.), (-1., 1.)]),
-    (ROT90, 1., 0., -TX, [(-1., 0.), (-1., 1.), (-2., 0.)]),
-    (ROT90, 1., 0., -TY, [(0., -1.), (0., 0.), (-1., -1.)]),
-    (ROT90, 1., 0., TXY, [(2., 3.), (2., 4.), (1., 3.)]),
-    (ROT180, 1., 0., T0, [(0., 0.), (-1., 0.), (0., -1.)]),
-    (ROT180, 1., 0., TX, [(1., 0.), (0., 0.), (1., -1.)]),
-    (ROT180, 1., 0., TY, [(0., 1.), (-1., 1.), (0., 0.)]),
-    (ROT180, 1., 0., -TX, [(-1., 0.), (-2., 0.), (-1., -1.)]),
-    (ROT180, 1., 0., -TY, [(0., -1.), (-1., -1.), (0., -2.)]),
-    (ROT180, 1., 0., TXY, [(2., 3.), (1., 3.), (2., 2.)]),
+    (0., (1., 1.), 0., T0, [(0., 0.), (1., 0.), (0., 1.)]),
```

Comment:
Shouldn't all the translations also be changed to tuples?

Reply:
They are tuples.... All these constants are of length 2. Yeah, so not easy to read.
Actually, SQ2 is a single value, and S23 is a tuple :-/